packages/components/ovh-shell/src/index.ts

```diff
@@ -1,7 +1,7 @@
 import DirectClientMessageBus from './message-bus/direct-client';
 import IFrameMessageBus from './message-bus/iframe';
 
-export { default as useShellClient } from './client';
+export { default as initShellClient } from './client';
```

Comment:
Seems to be a Breaking Change but here is no BREAKING CHANGE commit.

Reply:
Ok I updated the commit with a breaking change